src/cli/scaffold/rust.rs

```diff
@@ -36,6 +37,20 @@ impl Scaffold for RustScaffold {
             vec!["init".to_owned(), "--vcs".to_owned(), "none".to_owned()],
         )?;
 
+        // add hdk-rust dependency by default
+        let cargo_file_path = base_path.as_ref().join(package::CARGO_FILE_NAME);
+
+        let mut f = OpenOptions::new()
+            .append(true)
+            .open(cargo_file_path)?;
+
+        // @TODO switch to crates.io ref when hdk-rust gets published
+        // @see https://github.com/holochain/holochain-cmd/issues/19
+        let hdk_dep: &str = "hdk = { git = \"https://github.com/holochain/hdk-rust\" }";
+
+        f.write_all(hdk_dep.as_bytes())?;
```

Comment:
I would like to see the build configuration for wasm added as well. It needs to be built using this configuration:

```toml
[lib]
crate-type = ["cdylib"]
```

Reply:
@lucksus Do you know more about this?

Reply:
so you do think we should be generating these with '--lib' for crates of the library type then?

Reply:
@sphinxc0re can we add a ticket and address in a separate PR?

Reply:
I really think this should be done right here 😅

Reply:
Ok so I have questions: 
Will this crate-type property work if the crate is not of type `lib`?  I am happy to add it, but I'd like to understand what it does